tests/app/questionnaire/test_value_source_resolver.py

```diff
@@ -458,6 +458,45 @@ def test_response_metadata_source():
     )
 
 
+@pytest.mark.parametrize(
+    "list_item_id",
+    [None, "item-1"],
+)
+def test_calculated_summary_source(mocker, list_item_id):
```

Comment:
Source makes me think of the source of the calculation (i.e answers_to_calculate) rather than value_source, so  i would prefer test_calculated_summary_as_value_source/ test_calculated_summary_value_source